### PR TITLE
Easy Peasy

### PR DESCRIPTION
**Pull request by bmitchinson**
### Description: 
Closes issue #55 

You might need to attach `--build` to your normal docker command so that new npm dependencies are installed in the container. Ex: `docker-compose -f ~/repos/hdhs/dev-docker-compose.yml up --bulid`

### Notes:
- PatientInfoForm saves to central state upon clicking the submit button.
- Included a test for blood pressure validation to up coverage back to 100%
- `/easy-peasy` shows a debug component that reflects everything in easy-peasy for reference.

### Checklist:
- [x] 90% Test Coverage (100% coverage met)

### CODE REVIEW

File: pwa/src/state/enums.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/* istanbul ignore file */
+
+export enum SEX {
+    male = 'male',
+    female = 'female',
+}
+
+export enum Y_N {
```
Comment (abpwrs) [inline pwa/src/state/enums.ts:8]: What is our naming convention for enumerations? All caps? camelCase?

I'm not super partial, but the `_` in `Y_N` feels out of place.

Comment (bmitchinson) [reply pwa/src/state/enums.ts:8]: Yeah I'm not sure. Would be up for a rename. I prefer uppercase personally cause sometimes they're a nice way to distinguish from typescript interfaces.

`YN` ?

File: pwa/src/state/models/newPatientFormModels.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { Action, action } from 'easy-peasy'
+import { SEX, Y_N } from '../enums'
+
+interface PatientInfoFormFields {
+    firstName?: string
```
[the remark this answers is not in the document]
Comment (bmitchinson) [reply pwa/src/state/models/newPatientFormModels.ts:5]: Nope. Plan is to go through and reference what default responses should be in the event of adding to the database. Feel like to develop functionality we could make them optional for now, + then with the guidance of CHI disallow / allow various combinations of blank responses once we have submission in place.

File: pwa/src/state/models/newPatientFormModels.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    zScore?: number
+    albendazole?: Y_N
+    allergies?: string
+    medicines?: string
+    chiefComplaint?: string[]
+    history?: string
+    exam?: string
+}
+
+export interface PatientInfoFormModel {
```
[the remark this answers is not in the document]
Comment (bmitchinson) [reply pwa/src/state/models/newPatientFormModels.ts:34]: Yeah, I like that better than `Models`, renaming.

File: pwa/src/state/enums.ts (hunk 1 above)
Comment (bmitchinson) [reply pwa/src/state/enums.ts]: Changed to YN

**Pull request merged**